nav2_map_server/README.md

```diff
@@ -62,83 +110,157 @@ occupied_thresh: 0.65
 free_thresh: 0.196
 ```
 
+```
+$ map_server_3d map_3d.yaml
+```
+
+For 3D map server YAML can be:
+
+- for a `pcd` as input/output 
+```
+image: testmap.pcd
```

Comment:
@AlexeyMerzlyakov as we have changed the complete server, I think this tag should also be changed to something more relevant like `image` -> `pcd`.

Reply:
As I remember, we have already discussed it: `image` field is remaining for unification and backward compatibility with a 2D map YAML-formats. This indeed will represents better PCD, but will break the unification. It could rather lead to confusion in the future. So, I still consider to use unified `image` field is better than have separate field for each concrete type of data.

Reply:
Ok, then! marking this as resolved.